packages/replay/MIGRATION.md

```diff
@@ -52,3 +52,8 @@ import something from '@sentry/replay/submodule';
 ```
 
 If you only imported from `@sentry/replay`, this will not affect you.
+
+## Session object is now a POJO (https://github.com/getsentry/sentry-javascript/pull/6417)
+
+The `Session` object exported from Replay is now a POJO, instead of a class.
```

Comment:
also a nit - I would just say a plain object, might be easier for developers to understand (especially those who are more unfamiliar with JS).